source/indices.d.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Generates a tuple based on the length of the input array
+ * and fills each index with the index number.
+ *
```

Comment:
Remove the `*`. See the other types.

Reply:
that's neat. I like this style of comments,